index.js

```diff
@@ -675,7 +675,7 @@ Client.prototype.publisherInfo = function (publisher, callback) {
   })
 }
 
-Client.prototype.getPromotion = function (lang, callback) {
+Client.prototype.getPromotion = function (lang, callback, forPaymentId) {
```

Comment:
let's do it like `Client.prototype.getPromotion = function (lang, forPaymentId, callback) {`